jgrapht-core/src/test/java/org/jgrapht/alg/flow/TestPushRelabel.java

```diff
@@ -0,0 +1,34 @@
+package org.jgrapht.alg.flow;
+
+import junit.framework.TestCase;
+import org.jgrapht.alg.interfaces.MaximumFlowAlgorithm;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.SimpleDirectedGraph;
+
+/**
+ * Created by vernemat on 23/05/2017.
+ */
+public class TestPushRelabel extends TestCase {
```

Comment:
There is already a test case class called `PushRelabelMFImplTest`. You should place the test there.